docs/UserGuide.md

```diff
@@ -181,88 +165,90 @@ Format: **`help`**
+Alternate Format: **`addt`** `all t/TAG [t/TAG]…​`
+* Add tags to everyone in the displayed contacts list.
 * At least one `TAG` must be provided.
 * `TAG` is case-sensitive
+* Input tag will be appended to the existing tag.
+* If duplicate tags are entered, these tags will not be added.
 
 For example:
 
-* **`addt`** `all t/friend` followed by **`find`** `g/f` adds the friend tag to all persons in the filtered list of female contacts.
-* **`addt`** `all t/teammate t/neighbour` adds the teammate and neighbour tags to all persons in the displayed person list.
+* **`addt`** `all t/friend` after **`find`** `g/f` adds the `friend` tag to everyone in the filtered list of female contacts.
+* **`addt`** `all t/teammate t/neighbour` adds both `teammate` and `neighbour` tags to everyone in the displayed contacts list.
 ![result for 'addt all t/teammate t/neighbour'](images/AddTagsToAll.png)
 *After execution of Add Tag Command: **`addt`** `all t/teammate t/neighbour`*
 
-### Adding remark to a person: `remark`
+### Adding a remark to a person: `remark`
 
-You can add a remark to a person in the contact book by `INDEX`.
+You can add a remark to a person in your contacts list with the `remark` command.
 
-Format: **`remark`** `INDEX r/VALUE`
+Format: **`remark`** `INDEX r/REMARK`
 * Add a remark to the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* The index refers to the index number shown in the displayed contacts list.
 * Index must be provided and **must be a positive integer** 1, 2, 3, …​
 * Existing remark will be updated to the input remark.
-* If duplicated remarks are given, only the latest one will be taken.
-* You can remove a person’s remark by typing `remark` (i.e. empty parameter) or `remark r/` without specifying any remark after it.
+* If a duplicate remark is entered, only the latest remark will be taken.
+* You can remove a person’s remark by typing `remark` (i.e. empty parameter) or `remark r/` without specifying any remark.
 
 Examples:
 
-* **`remark`** `1 r/She likes coding` adds the remark `She likes coding` to the person at index 1.
-* **`remark`** `2` clears the remark of the person at index 2.
+* **`remark`** `1 r/She likes coding` adds the remark `She likes coding` to first person in the displayed contacts list.
+* **`remark`** `2` clears the remark of the second person in the displayed contacts list.
 ![result for 'remark 2'](images/RemoveRemark.png)
 *After execution of Remark Command: **`remark`** `2`*
 
 ### Editing a person: `edit`
 
-You can edit an existing person in the contact book by `INDEX`.
+You can edit an existing person in your contacts list with the `edit` command.
 
-Format: **`edit`** `INDEX FIELD_PREFIX/VALUE [FIELD_PREFIX/VALUE]…​`
+Format: **`edit`** `INDEX PREFIX/KEYWORD [PREFIX/KEYWORD]…​`
 
-* `FIELD_PREFIX/VALUE` can be any of the following:
+* `PREFIX/KEYWORD` can be substituted with any of the following:
```

Comment:
Why is it called `KEYWORD`?

Reply:
Sounds weird to me on hindsight. Reverted back to VALUE